src/rewriters/CounterRewriter.h

```diff
@@ -0,0 +1,69 @@
+//
+// Created by Martin Blicha on 03.02.21.
+//
+
+#ifndef OPENSMT_DIVMODREWRITER_H
+#endif
+
+#include "PTRef.h"
+
+#include "Rewriter.h"
+#include "ArithLogic.h"
+
+#include "OsmtApiException.h"
+
+#include <unordered_map>
+#include <string>
+
+class CounterConfig : public DefaultRewriterConfig {
```

Comment:
It seems to me that this whole class should be a `Visitor`, not a `Rewriter`, no?

Reply:
It's removed cause Antti said that it's not the thing to have in master